gammapy/background/models.py

```diff
@@ -71,3 +81,825 @@ def evaluate(self, x, y):
+            name of file with the bg cube
+
+        Returns
+        -------
+        bg_cube : `~gammapy.models.CubeBackgroundModel`
+            bg model cube object
+        """
+        hdu = fits.open(filename)
+        return CubeBackgroundModel.from_fits_bin_table(hdu)
+
+    @staticmethod
+    def read_image(filename):
+        """Read cube background model from image in fits file.
+
+        Parameters
+        ----------
+        filename : `~string`
+            name of file with the bg cube
+
+        Returns
+        -------
+        bg_cube : `~gammapy.models.CubeBackgroundModel`
+            bg model cube object
+        """
+        hdu = fits.open(filename)
+        return CubeBackgroundModel.from_fits_image(hdu)
+
+    def to_astropy_table(self):
+        """Convert cube background model to astropy table format.
+
+        Returns
+        -------
+        name : `~string`
+            name of the table
+        table : `~astropy..table.Table`
+            table containing the bg cube
+        """
+        # fits unit string
+        u_detx = '{0.unit:FITS}'.format(self.detx_bins)
+        u_dety = '{0.unit:FITS}'.format(self.dety_bins)
+        u_energy = '{0.unit:FITS}'.format(self.energy_bins)
+        u_bg = '{0.unit:FITS}'.format(self.background)
+
+        # data arrays
+        a_detx_lo = Quantity([self.detx_bins[:-1]])
+        a_detx_hi = Quantity([self.detx_bins[1:]])
+        a_dety_lo = Quantity([self.dety_bins[:-1]])
+        a_dety_hi = Quantity([self.dety_bins[1:]])
+        a_energy_lo = Quantity([self.energy_bins[:-1]])
+        a_energy_hi = Quantity([self.energy_bins[1:]])
+        a_bg = Quantity([self.background])
+
+        # name
+        name = 'BACKGROUND'
+        # TODO: is it possible to give a name to a `~astropy.table.Table`?
+        #       (without writing it in the header)
+
+        # table
+        table = Table()
+        table['DETX_LO'] = a_detx_lo
+        table['DETX_HI'] = a_detx_hi
+        table['DETY_LO'] = a_dety_lo
+        table['DETY_HI'] = a_dety_hi
+        table['ENERG_LO'] = a_energy_lo
+        table['ENERG_HI'] = a_energy_hi
+        table['Bgd'] = a_bg
+
+        table.meta['E_THRES'] = a_energy_lo.flatten()[0].value
+
+        return name, table
+
+    def to_fits_bin_table(self):
+        """Convert cube background model to binary table fits format.
+
+        Returns
+        -------
+        tbhdu : `~astropy.io.fits.BinTableHDU`
+            table containing the bg cube
+        """
+        # build astropy table
+        name, table = self.to_astropy_table()
+
+        data = table.as_array()
+
+        header = fits.Header()
+        header.update(table.meta)
+
+        tbhdu = fits.BinTableHDU(data, header, name=name)
+
+        # Copy over column meta-data
+        for colname in table.colnames:
+            tbhdu.columns[colname].unit = str(table[colname].unit)
+
+        # TODO: this method works fine but the order of keywords in the table
```

Comment:
Remove this TODO. Make a Github issue instead. This affects other FITS I/O methods as well.